script.js

```diff
@@ -0,0 +1,108 @@
+const buttonLogin = document.getElementById('button-login');
+const inputName = document.getElementById('input-login');
+const inputPassword = document.getElementById('input-password');
+const checkboxAgreement = document.getElementById('agreement');
+const userName = 'tryber@teste.com';
+const password = '123456';
```

Comment:
O que acha das credenciais de login ser um objeto? 🤔

```suggestion
const credentials = {
  userName: 'tryber@teste.com',
  password: '123456'
}
```

O que acha? ;)